charts/metallb/values.yaml

```diff
@@ -6,6 +6,8 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 loadBalancerClass: ""
+# ignore the exclude-from-external-loadbalancer annotation
+ignoreExcludeLB: false
```

Comment:
Would it be better to put it on speaker?

Reply:
My reasoning was, this is a property of metallb as a whole. It has direct effects only on the speaker, but it felt more like a property similar to "loadBalancerClass". Does it make sense? If not I am open to move it

Reply:
I think we already have a similar configuration, see https://github.com/metallb/metallb/blob/d9ab5ec6f5a48988ef67c314e590f01e61c2c1c5/charts/metallb/values.yaml#L269

It's more of a speaker-related property, but I'm ok with both, it's up to you :)

nit: the comment should be updated: `ignore the exclude-from-external-loadbalancer label`

Reply:
Fair enough, I'll change it.
Also, thanks for the comment, I missed it!

Reply:
done